modules/integration/docker/src/test/scala/scala/cli/integration/RunDockerTests.scala

```diff
@@ -42,4 +42,21 @@ class RunDockerTests extends munit.FunSuite {
       expect(output.linesIterator.toVector.last == message)
     }
   }
+
+  test("package simple app with native in docker") {
+    val fileName = "simple.sc"
+    val inputs = TestInputs(
+      os.rel / fileName -> """println("Hello")"""
+    )
+    inputs.fromRoot { root =>
+      val cmdPackage = Seq[os.Shellable](
+        // format: off
+        "docker", "run", "--rm", termOpt, "-v", s"$root:/data", "-w", "/data", ciOpt,
+        imageName, "package", "--native", fileName, "-o", "Hello"
+        // format: on
+      )
+      val procPackage = os.proc(cmdPackage).call(cwd = root)
+      expect(procPackage.exitCode == 0)
```

Comment:
is this line necessary? if the `exitCode` wasn't 0, the test would fail, since there's no `check = false`, no?